grafana-plugin/src/state/rootBaseStore.ts

```diff
@@ -239,7 +234,9 @@ export class RootBaseStore {
   }
 
   isUserActionAllowed(action: UserAction) {
-    return this.userStore.currentUser && this.userStore.currentUser.permissions.includes(action);
+    // https://github.com/grafana/grafana/blob/main/public/app/core/services/context_srv.ts#L165-L170
+    // TODO: what to assign fallback (2nd argument) to?
+    return contextSrv.hasAccess(action, true);
```

Comment:
this is important

Reply:
@gamab how're you currently handling the `fallback` arg to `contextSrv.hasAccess`?